src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -884,4 +887,10 @@ public void verifyAdChainForSlot(String[] creativeIdChain, String slotName, AdsB
   public By findFirstIframeWithAd(String slotName) {
     return By.cssSelector("#" + slotName + " iframe[title='3rd party ad content']");
   }
+
+  public void waitForVASTRequestWithAdUnit(NetworkTrafficInterceptor networkTrafficInterceptor, String adUnit) throws UnsupportedEncodingException {
+    final String encodedAdUnit = URLEncoder.encode(adUnit, "UTF-8");
+    final String PATTERN = ".*output=xml_vast.*iu=" + encodedAdUnit + ".*";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)